plugins/versionpress/tests/Automation/WpAutomationRunner.sample.php

```diff
@@ -28,6 +30,46 @@ public function runAutomation()
     /**
```

Comment:
Please add link to `versionpress/pedestal` repository. Users/developers may not be familiar with it.